go/mysql/client.go

```diff
@@ -640,7 +721,7 @@ func (c *Conn) writeHandshakeResponse41(capabilities uint32, scrambledPassword [
 	}
 
 	// Assume native client during response
```

Comment:
I realize this is copied from upstream, so maybe not to fix, but this comment looks stale...

Reply:
Good call; that's the current comment in the tip of `vitessio/vitess`, but it does seem clearly wrong. I updated it to a more sane comment.